.github/workflows/lint.yml

```diff
@@ -318,6 +318,17 @@ jobs:
             echo 'Please fix the above Flake8 warnings.'
             false
           fi
+      - name: Add annotations
+        # Don't run on forked pull requests
+        if: ${{ failure() && github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
This will also ensure that this won't run on trunk since `github.event.pull_request.head.repo.full_name` will equal out to `''` if not present